docs/how-tos/create-w3c-document.md

```diff
@@ -0,0 +1,483 @@
+    let provider;
+    if (ethers.version.startsWith("6.")) {
+      provider = new (ethers as any).JsonRpcProvider(CHAININFO.rpcUrl);
+    } else if (ethers.version.includes("/5.")) {
+      provider = new (ethers as any).providers.JsonRpcProvider(
+        CHAININFO.rpcUrl
+      );
+    }
+    const signer = unconnectedWallet.connect(provider);
+
+    let transactionOptions = {};
+    const { gasStation } = CHAININFO;
+    if (gasStation) {
+      try {
+        const gasFees = await gasStation();
+        const { maxFeePerGas, maxPriorityFeePerGas } = gasFees ?? {};
+        transactionOptions = {
+          maxFeePerGas: maxFeePerGas?.toString() ?? 0,
+          maxPriorityFeePerGas: maxPriorityFeePerGas?.toString() ?? 0,
+        };
+      } catch (error) {
+        console.warn("Failed to fetch gas prices", error);
+      }
+    }
+
+    console.log("Minting document...");
+    let tx = await mint(
+      {
+        tokenRegistryAddress: TOKEN_REGISTRY_ADDRESS,
+      },
+      signer,
+      {
+        beneficiaryAddress,
+        holderAddress,
+        tokenId,
+        remarks,
+      },
+      {
+        id: signedW3CDocument?.id!,
+        ...transactionOptions,
+      }
+    );
+
+    console.log("Transaction submitted successfully", tx.hash);
+    const receipt = await tx.wait();
+    return receipt;
+  } catch (error) {
+    console.error(error);
+  }
+};
+```
+
+### 8. Verifying the Document
+
+After creating and signing your W3C document, you can verify its authenticity and integrity. The verification process returns an array of verification fragments, each representing a different aspect of the document's validity.
+
+Create a new file `src/utils/verifyDocument.ts` with the verification logic:
+
+```typescript
+require("dotenv").config();
+import {
+  verifyDocument,
+  SignedVerifiableCredential,
+  CHAIN_ID,
+  SUPPORTED_CHAINS,
+} from "@trustvc/trustvc";
+
+export const verifyW3CDocument = async (
+  signedDocument: SignedVerifiableCredential
+) => {
+  try {
+    console.log("Starting document verification...");
+    const CHAINID: CHAIN_ID = (process.env.NET as CHAIN_ID) ?? CHAIN_ID.amoy;
+    const CHAININFO = SUPPORTED_CHAINS[CHAINID];
+    const RPC_PROVIDER_URL = CHAININFO.rpcUrl;
+
+    const verificationResult = await verifyDocument(signedDocument, {
+      rpcProviderUrl: RPC_PROVIDER_URL,
+    });
+
+    return verificationResult;
+  } catch (error) {
+    console.error("Error verifying document:", error);
+    throw error;
+  }
+};
+
+```
+
+The verification process checks multiple aspects of your document. To understand the different components of verification and what each fragment represents, refer to the [TradeTrust verification overview](https://docs.tradetrust.io/docs/introduction/key-components-of-tradetrust/w3c-vc/verifying-documents/overview), and the different [W3C verification fragments](https://docs.tradetrust.io/docs/tutorial/verifier#w3c-verifiers)
```

Comment:
update the link to use internal routing:
/docs/introduction/key-components-of-tradetrust/w3c-vc/verifying-documents/overview/
/docs/tutorial/verifier/#w3c-verifiers